Website/src/rendering/src/services/SendService.tsx

```diff
@@ -0,0 +1,31 @@
+import Script from 'next/script';
+
+const SEND_WEBSITE_ID = process.env.NEXT_PUBLIC_SEND_WEBSITE_ID || '';
```

Comment:
`NEXT_PUBLIC_SEND_WEBSITE_ID` is missing from `docker-compose.override.yml`. Also, `SEND_WEBSITE_ID=` is missing from `.env`. I had previously started branch `feature/send-integration` with boilerplate code from the env file up to the SendService for Antonis to work on this integration. Maybe we could take the missing pieces from it.